Week 2 - Arrays/scrabble/scrabble.c

```diff
@@ -0,0 +1,72 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <stdio.h>
+
+int getIndex(char key);
+int getValue(char key);
+int countPoints(string word);
+
+int SIZE = 26;
+char KEYS[26] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
```

Comment:
No need to specify size of array for literals
```suggestion
char KEYS[] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
```